### PR TITLE
Handle nulls on leaderboard

### LINKED ISSUE #603: Leaderboard values are not displayed in case of any occurence null values
**Describe the bug**
Leaderboard values are not displayed in case of any occurence null values

**To Reproduce**
1. Import nyc311 database
2. Create a measure with expression `avg(latitude)`
3. Choose formatting as Decimal
4. Go to dashboard

### PR DESCRIPTION
**Pull request by djbarnwal**
Closes #603 

After fix - 

https://user-images.githubusercontent.com/4402679/179972604-941d8958-ed36-49fe-b2f7-5e338bf0857e.mp4

### CODE REVIEW

File: src/common/database-service/DatabaseMetricsExploreActions.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ export class DatabaseMetricsExploreActions extends DatabaseActions {
       SELECT ${expression} as value, "${column}" as label from "${table}"
       WHERE "${column}" IS NOT NULL ${whereClause}
       GROUP BY "${column}"
-      ORDER BY value desc
+      ORDER BY value desc NULLS LAST
```
Comment (AdityaHegde) [inline src/common/database-service/DatabaseMetricsExploreActions.ts:44]: If we are filtering out nulls then we should do that here.

Comment (hamilton) [reply src/common/database-service/DatabaseMetricsExploreActions.ts:44]: yeah, maybe in hindsight filtering out is the better move

Comment (djbarnwal) [reply src/common/database-service/DatabaseMetricsExploreActions.ts:44]: Maybe we can show the null labels but make them un-selectable? That way there is no loss of information.

**Review by AdityaHegde — APPROVED** [review]

**Comment by djbarnwal** [comment]
Merging as being blocked by it. We can address if we want to get rid of nulls or not in a follow up PR later

**Pull request merged**